src/styles/main.scss

```diff
@@ -0,0 +1,6 @@
+@import 'variables';
+@import 'header';
+@import 'index';
+@import 'stars';
+@import 'card';
+@import 'reset';
```

Comment:
The 'reset' import should typically be the first import in your main stylesheet to ensure that all styles are reset before any other styles are applied. Consider moving the '@import 'reset';' to the top of the file.